### ISSUE TITLE
3dfx Splash Screen won´t show up

### ISSUE THREAD

**Issue by Starfoxfs** [joncampbell123/dosbox-x#2442]
I have copied 3dfxspl2.dll and 3dfxspl3.dll from dgVoodoo2 into Dosbox-X Directory
and set in dosbox-x.conf splash=true .

No 3dfx Logo shows up for Dos Games.

Dosbox Version 0.83.13 Mingw 64bit SDL2

**Comment by Wengier** [comment]
I think this is probably caused by a mismatch of 32-bit DLL and 64-bit DOSBox-X build. Since you use 64-bit DOSBox-X a 64-bit 3dfxspl2.dll is required for this to work, and vice versa.

**Comment by Starfoxfs** [comment]
Thanks that was the reason, 32bit works

**Comment by isidoro-80** [comment]
> I have copied 3dfxspl2.dll and 3dfxspl3.dll from dgVoodoo2 into Dosbox-X Directory
> and set in dosbox-x.conf splash=true .
> 
> No 3dfx Logo shows up for Dos Games.
> 
> Dosbox Version 0.83.13 Mingw 64bit SDL2


hi i would like no vodoo drivers in windows 98 can you help me?

**Comment by Wengier** [comment]
@isidoro-80 I guess this is a different issue from the original issue in this thread. Perhaps you can make a separate thread for this and also try to describe your problem more clearly. Thanks!

**Comment by rderooy** [comment]
I agree with what @Wengier said, but in addition you should also check out the wiki.
https://dosbox-x.com/wiki/Guide%3ASetting-up-3dfx-Voodoo-in-DOSBox%E2%80%90X
https://dosbox-x.com/wiki/Guide%3AInstalling-Windows-98
